src/AstroBase.jl

```diff
@@ -1,5 +1,22 @@
 module AstroBase
 
-# package code goes here
+export a00, anp
+const J2000 = 2451545.0
+anp = x -> mod(x, 2pi) < 0? mod(x, 2pi)+2pi:mod(x, 2pi)
```

Comment:
Assigning an anonymous function in global scope like this is probably a bad idea. Apart from the fact that it is not needed (see my other comment), why not make it a proper function? `anp(x) = mod(x, 2pi) < 0? mod(x, 2pi)+2pi:mod(x, 2pi)`